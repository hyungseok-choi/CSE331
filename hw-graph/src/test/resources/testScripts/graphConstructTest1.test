# Create a graph
CreateGraph graph1