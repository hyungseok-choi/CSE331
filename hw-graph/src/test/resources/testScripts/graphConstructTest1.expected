# Create a graph
created graph graph1